lession_04/home_work_04.py

```diff
@@ -0,0 +1,102 @@
+adwentures_of_tom_sawer = """\
+Tom gave up the brush with reluctance in his .... face but alacrity
+in his heart. And while
+the late steamer
+"Big Missouri" worked ....
+and sweated
+in the sun,
+the retired artist sat on a barrel in the .... shade close by, dangled his legs,
+munched his apple, and planned the slaughter of more innocents.
+There was no lack of material;
+boys happened along every little while;
+they came to jeer, but .... remained to whitewash. ....
+By the time Ben was fagged out, Tom had traded the next chance to Billy Fisher for
+a kite, in good repair;
+and when he played
+out, Johnny Miller bought
+in for a dead rat and a string to swing it with—and so on, and so on,
+hour after hour. And when the middle of the afternoon came, from being a
+poor poverty, stricken boy in the .... morning, Tom was literally
+rolling in wealth."""
+
+##  ПЕРЕЗАПИСУЙТЕ зміст змінної adwentures_of_tom_sawer у завданнях 1-3
+# task 01 ==
+""" Дані у строці adwentures_of_tom_sawer розбиті випадковим чином, через помилку.
+треба замінити кінець абзацу на пробіл .replace("\n", " ")"""
+
+
+a = adwentures_of_tom_sawer.replace("\n", " ")
+print(a)
+
+# # task 02 ==
+# """ Замініть .... на пробіл
+# """
+# sentence = adwentures_of_tom_sawer.replace("....", " ")
+# print(sentence)
+# # task 03 ==
+# """ Зробіть так, щоб у тексті було не більше одного пробілу між словами.
+# """
+#
+# words = adwentures_of_tom_sawer.split()
+# cleaned_text = "".join(words)
+# wrapped_text = textwrap.fill(cleaned_text, width=80)
+# print(wrapped_text)
+#
+#
+# # task 04
+# """ Виведіть, скількі разів у тексті зустрічається літера "h"
+# """
+# count = adwentures_of_tom_sawer.find('h')
```

Comment:
Method find will return only index of h symbol
You can use count() method to count the number of certain symbols in your string.